novawallet/Common/Extension/Foundation/String+Truncation.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+extension String {
+    func truncatedMiddle(
```

Comment:
We have the same logic in `extension AccountAddress`. I would rather rename it to `extension String` as AccountAddress is a type alias to String